package-lock.json

```diff
@@ -12423,7 +12423,7 @@
     },
     "through": {
       "version": "2.3.8",
-      "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
+      "resolved": "http://registry.npmjs.org/through/-/through-2.3.8.tgz",
```

Comment:
of the cryptic and superfluous things that NPM seems to do, this one is by far my least favorite.  i mean, i know there's a checksum there, and presumably/hopefully it's being used for integrity checking.  but any `http` -> `https` still bugs me.